corehq/apps/registration/utils.py

```diff
@@ -51,15 +51,39 @@
 )
 
 
-def activate_new_user(form, created_by, created_via, is_domain_admin=True, domain=None, ip=None):
-    username = form.cleaned_data['email']
-    password = form.cleaned_data['password']
+def activate_new_user_via_reg_form(form, created_by, created_via, is_domain_admin=True, domain=None, ip=None):
     full_name = form.cleaned_data['full_name']
+    new_user = activate_new_user(
+        username=form.cleaned_data['email'],
+        password=form.cleaned_data['password'],
+        created_by=created_by,
+        created_via=created_via,
+        first_name=full_name[0],
+        last_name=full_name[1],
```

Comment:
I see this didn't change here, but is `full_name` always a two-element sequence? How about for names like _Guido van Rossum_?

Reply:
yes https://github.com/dimagi/commcare-hq/blob/c9280cf3acff22e72ae7dc67b73d53db9f4dc831/corehq/apps/registration/forms.py#L217-L219